src/libraries/Common/src/System/LocalAppContextSwitches.Common.cs

```diff
@@ -56,6 +56,11 @@ private static bool GetSwitchDefaultValue(string switchName)
                 return true;
             }
 
+            if (switchName == "Switch.System.Diagnostics.StackTrace.ILOffsetToStackTrace")
+            {
+                return Environment.GetEnvironmentVariable("COMPlus_ILOffsetToStackTrace") == "1" ? true : false;
```

Comment:
How about using ```Equals``` method?
```
return Environment.GetEnvironmentVariable("COMPlus_ILOffsetToStackTrace").Equals("1");
```

Reply:
If *COMPlus_ILOffsetToStackTrace* is not set, *GetEnvironmentVariable()* method returns *null*.
As a result, *NullReferenceException* can occur.
So, it is not proper to use *Equals()* method